docs/modules/ROOT/pages/explanations/commodore-components/container-images.adoc

```diff
@@ -0,0 +1,33 @@
+= Managing container image versions
+
+It's considered best practice to provide component parameter `images` which contains the default container image versions for all images used by the component.
+
+NOTE: This structure may become mandatory to support automated component dependency updates using Renovate.
```

Comment:
When I discussed this feature with Simon Rüegg, we thought that this requirement shouldn't be. It should be possible for Renovate to detect any images in the parameter structure, if they follow a certain convention, e.g. the keyword `image` followed by at least `repository` and `tag` maybe.
```yaml
parameters:
  someComponent:
    someOtherKey:
      image:
        registry: docker.io
        repository: company/imagename
        tag: latest
```
The reason to lift this limitation is for example when Helm values are exposed: Renovate can then make PRs to update those as well. Also we'd avoid having to duplicate the image in `parameters.images` and then reference via helm values like `parameters.helm_values.image.repository: ${mycomponent:images:someimage:repository}`